_posts/2018-09-12-september-office-hours.md

```diff
@@ -0,0 +1,72 @@
+---
+title: September Office Hours
+description: Multi-user OpenFaaS Cloud, Istio Service Mesh, and Stateless Microservices
+
+date: 2018-09-11
+image: /images/september-office-hours/team.jpg
+categories:
+  - office-hours
+  - community
+author_staff_member: burton
+dark_background: true
+---
+
```

Comment:
True (keep it in as a generic statement) but highlight the content of this particular call's content. This is the text that you should imagine being in a Twitter card etc.